android/assets/jsons/translations/Russian.properties

```diff
@@ -935,9 +933,9 @@ Paper Maker = Бумажная мастерская
 Circus = Цирк
 
  # Requires translation!
-Must be next to river = 
+Must be next to river = Должно быть рядом с водой
```

Comment:
What must be next to river, city?

Reply:
To provide more context on why I ask it, Russian is a gendered language so depending on the item sentence would look different (e.g. "city" is masculine, "building" is neuter and "farm" is feminine, so it'd be different for these 3).

Reply:
Building